docs/semgrep-code/overview.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: overview
+append_help_link: true
+title: Overview
+hide_title: false
+description: Learn about Code, a static application security testing (SAST) tool |
+    designed to find security vulnerabilities in your applications.
+tags:
+  - code
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import SemgrepScan from "/src/components/concept/_semgrep-scan.mdx"
+
+Semgrep Code is a static application security testing (SAST) tool that inspects your application for security vulnerabilities.
+You can use Semgrep Code to scan local repositories or integrate it into your CI/CD pipeline to automate the continuous scanning of your code.
+
+## Rules
+
+<SemgrepScan />
+
+In addition to the default rules (available in the [Registry](https://semgrep.dev/r)), you can write custom rules to determine how Semgrep Code scans your repositories. Whether you use the default rules or write custom rules, knowing *which* rules Semgrep Code ran helps you understand how it detected security issues.
```

Comment:
```suggestion
In addition to rules available in the [Registry](https://semgrep.dev/r), you can write custom rules to determine what Semgrep Code detects in your repositories. Whether you use pre-existing rules or write custom rules, knowing *which* rules Semgrep Code runs can help you understand how it detects security issues.
```

- The rules in the Registry are not actually "default" - a lot of them are community contributed, some of them are from our SR team, so the quality and freshness can vary. Then there is also the "default" ruleset, which is something else entirely, so I just said more or less that the Registry is where they can find rules.
- There's no particular time that the scan occurred (we're not in a procedure) so I kept it to present tense.